src/rdb/tests/rpc.h

```diff
@@ -74,12 +113,75 @@ CRT_RPC_DECLARE(rdbt_init, DAOS_ISEQ_RDBT_INIT_OP, DAOS_OSEQ_RDBT_INIT_OP)
 
 CRT_RPC_DECLARE(rdbt_fini, DAOS_ISEQ_RDBT_FINI_OP, DAOS_OSEQ_RDBT_FINI_OP)
 
+#define DAOS_ISEQ_RDBT_PING_OP	/* input fields (none) */
+
+#define DAOS_OSEQ_RDBT_PING_OP	/* output fields */		 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
False positive for CaRT RPC definition related macros